### ISSUE TITLE
Selection of super parameters such as learning rate and weight decay

### ISSUE THREAD

**Issue by JonasZero** [thuml/Transfer-Learning-Library#125]
Hi, 
I found that the settings of learning rate, weight decay and lr scheduler of domain adaptation method are inconsistent.  Is it necessary to set the same training parameters for different methods? And does the benchmark of different task algorithms provided in the document follow the same training parameters?

**Comment by JunguangJiang** [comment]
It's not necessary to set the same training parameters for different methods.

**Comment by JonasZero** [comment]
> It's not necessary to set the same training parameters for different methods.

Okay, thank you.
